### PR TITLE
fix: fix http server start

### PR DESCRIPTION
**Pull request by ZuLiangWang**
# Which issue does this PR close?

Closes #

# Rationale for this change
Now the http server will be started when the meta starts synchronously, this will cause the meta server to fail to initialize normally.

# What changes are included in this PR?
* Change the start of http server from synchronous to asynchronous.

# Are there any user-facing changes?
None.

# How does this change test
Local manual test.

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -184,10 +184,14 @@ func (srv *Server) startServer(_ context.Context) error {
...
 	api := http.NewAPI(procedureManager, procedureFactory, manager)
 	httpService := http.NewHTTPService(srv.cfg.HTTPPort, time.Second*10, time.Second*10, api.NewAPIRouter())
-	err = httpService.Start()
-	if err != nil {
-		return errors.WithMessage(err, "start http service failed")
-	}
+	go func() {
+		err := httpService.Start()
+		if err != nil {
+			if err != nil {
```
Comment (ShiKaiWi) [inline server/server.go:190]: Remove this redundant check.

File: server/server.go [hunk 2]
```diff
@@ -184,10 +184,14 @@ func (srv *Server) startServer(_ context.Context) error {
 
 	api := http.NewAPI(procedureManager, procedureFactory, manager)
 	httpService := http.NewHTTPService(srv.cfg.HTTPPort, time.Second*10, time.Second*10, api.NewAPIRouter())
-	err = httpService.Start()
-	if err != nil {
-		return errors.WithMessage(err, "start http service failed")
-	}
+	go func() {
+		err := httpService.Start()
```
Comment (ShiKaiWi) [inline server/server.go:188]: Any control method to close the http service?

Comment (ZuLiangWang) [reply server/server.go:188]: `http.Stop()` has been definition but not used. At present, the service does not need to actively shut down the http server.

**Review by ShiKaiWi — APPROVED** [review]
LGTM

**Pull request merged**